execution_workflows/DaPars2/Dapars2_snakemake/config/config.DaPars2.yaml

```diff
@@ -0,0 +1,21 @@
+# Configuration file for snakemake pipeline
+
+# Path to run-specific sample table of | sample | bam
+samples: "config/samples.csv"
+
+# Path to main output directory for run
+out_dir: "results/"
+
+# Path to GTF annotation file. Note chromosome names SHOULD NOT have a 'chr' prefix
+gtf: "../../../tests/test_data/gencode_2genes_noChr.vM18.annotation.gtf"
```

Comment:
I'm not sure why two versions of the same genome annotation are necessary. They should be compatible to each other.